app/views/agent/CheckYourAnswersView.scala.html

```diff
@@ -50,8 +51,10 @@ <h2 class="govuk-heading-m">@messages("checkYourAnswers.vatBusinessDetails")</h2
         )
     ))
 
-    @govukButton(Button(
-      content = Text(messages("common.confirmAndContinue")),
-      href = Some(controllers.agent.routes.ConfirmEmailController.isEmailVerified.url)
-    ))
+    @formWithCSRF(action = controllers.agent.routes.ConfirmEmailController.isEmailVerified) {
```

Comment:
The work all looks good, but can you add in unit tests to verify the form action and the double click attribute please